CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 ### Changed
 - Breaking: Change `winapi` dependency to `windows-sys`. This is a breaking change since some of the low level Windows types are exposed in the public API of this library.
+- Breaking: Update `widestring` dependency from version 0.5.1 to 1. This is a breaking change since `NulError` (now `widestring::error::ContainsNul`) is exposed in the public API of this library.
```

Comment:
This is not exactly true. Since https://github.com/mullvad/windows-service-rs/pull/66 widestring is not part of the public API anymore.

So since 0.4.0, there is indeed a breaking change, but it is to vendor this NulError and no longer expose widestring in the public API (see the previously mentioned PR), it isn't the update of widestring to 1.0

Reply:
Yes and no. Since `windows-service 0.4.0` upgrading `widestring` and changing the error type, is a breaking change. So it should be mentioned in the changelog. BUT the upgrade from 0.5 -> 1.0 is not the breaking part. But I can fix that. I need to polish the changelog anyway.